autogluon/utils/tabular/ml/learner/abstract_learner.py

```diff
@@ -452,6 +452,23 @@ def leaderboard(self, X=None, y=None, silent=False):
                 print(leaderboard)
         return leaderboard
 
+    # TODO: Should this return a dictionary or pandas Series? Current it returns a Series
+    # TODO: enable_fit_continuation must be set to True to be able to pass X and y as None in this function, otherwise it will error.
+    # Warning: This can take a very, very long time to compute if the data is large and the model is complex.
+    # TODO: Warning: This is not memory safe, large datasets may result in OOM errors during feature importance calculation.
+    # A value of 0.01 means that the objective metric error would be expected to increase by 0.01 if the feature were removed.
```

Comment:
I don't think this is how these values should be explained in the eventual documentation, because they're often a poor substitute for remove-and-retrain values.  Rather, I think they're more appropriately described as how much the model depends on the feature (A value of 0.01 indicates the metric error is expected to increase by 0.01 when the information regarding this feature's value is hidden from the model).

Reply:
Agreed, will clean the descriptions when surfaced to users.